docs/ref/templates/builtins.txt

```diff
@@ -842,13 +842,13 @@ output (as a string) inside a variable. This is useful if you want to use
 
 Regroups a list of alike objects by a common attribute.
 
-This complex tag is best illustrated by way of an example: say that "places" is
-a list of cities represented by dictionaries containing ``"name"``,
+This complex tag is best illustrated by way of an example: say that ``places``
+is a list of cities represented by dictionaries containing ``"name"``,
 ``"population"``, and ``"country"`` keys:
 
 .. code-block:: python
 
-    cities = [
+    places = [
```

Comment:
Maybe something like `{% for country, countrys_cities in regrouped %}` could help with that.